submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/SubmissionReleaseTest.kt

```diff
@@ -0,0 +1,192 @@
+package ac.uk.ebi.biostd.itest.test.submission.submit
+
+import ac.uk.ebi.biostd.client.integration.commons.SubmissionFormat.TSV
+import ac.uk.ebi.biostd.client.integration.web.BioWebClient
+import ac.uk.ebi.biostd.itest.common.SecurityTestService
+import ac.uk.ebi.biostd.itest.entities.SuperUser
+import ac.uk.ebi.biostd.itest.itest.ITestListener.Companion.ftpPath
+import ac.uk.ebi.biostd.itest.itest.ITestListener.Companion.submissionPath
+import ac.uk.ebi.biostd.itest.itest.ITestListener.Companion.tempFolder
+import ac.uk.ebi.biostd.itest.itest.getWebClient
+import ac.uk.ebi.biostd.persistence.common.service.SubmissionPersistenceQueryService
+import ac.uk.ebi.biostd.submission.config.FilePersistenceConfig
+import ebi.ac.uk.asserts.assertThat
+import ebi.ac.uk.dsl.tsv.line
+import ebi.ac.uk.dsl.tsv.tsv
+import ebi.ac.uk.extended.mapping.to.ToSubmissionMapper
+import ebi.ac.uk.io.FileUtils
+import ebi.ac.uk.io.ext.createFile
+import ebi.ac.uk.util.date.toStringDate
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.runTest
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.condition.EnabledIfSystemProperties
+import org.junit.jupiter.api.condition.EnabledIfSystemProperty
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT
+import org.springframework.boot.test.web.server.LocalServerPort
+import org.springframework.context.annotation.Import
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import java.io.File
+import java.time.OffsetDateTime
+
+@Import(FilePersistenceConfig::class)
+@ExtendWith(SpringExtension::class)
+@SpringBootTest(webEnvironment = RANDOM_PORT)
+class SubmissionReleaseTest(
+    @Autowired val securityTestService: SecurityTestService,
+    @Autowired val submissionRepository: SubmissionPersistenceQueryService,
+    @Autowired val toSubmissionMapper: ToSubmissionMapper,
+    @LocalServerPort val serverPort: Int,
+) {
+    private lateinit var webClient: BioWebClient
+
+    @BeforeAll
+    fun init(): Unit = runBlocking {
+        securityTestService.ensureUserRegistration(SuperUser)
+        webClient = getWebClient(serverPort, SuperUser)
+    }
+
+    @Test
+    @EnabledIfSystemProperties(
+        value = [
+            EnabledIfSystemProperty(named = "enableFire", matches = "false"),
+            EnabledIfSystemProperty(named = "includeSecretKey", matches = "false"),
+            EnabledIfSystemProperty(named = "nfsReleaseMode", matches = "HARD_LINKS"),
+        ]
+    )
+    fun `27-1 public submission without secret key and HARD_LINKS release mode`() = runTest {
```

Comment:
I think rather add all this properties and conditions in test lets just simple always execute this test as part of normal workflows. Check SubmitWebBasePath

```
    @SpringBootTest(webEnvironment = RANDOM_PORT, properties = ["app.subBasePath=base/path"])
    inner class SubmitWebBasePath(@LocalServerPort val serverPort: Int) {
```

Reply:
Really good catch